src/Utopia/Messaging/Adapter/SMS/Fast2SMS.php

```diff
@@ -113,4 +115,24 @@ protected function process(SMSMessage $message): array
 
         return $response->toArray();
     }
+
+    /**
+     * Removes country code from a phone number
+     * Fast2SMS expects Indian phone numbers without the country code
+     *
+     * @param string $number Phone number with potential country code
+     * @return string Phone number without country code
+     */
+    private function removeCountryCode(string $number): string
+    {
+        // Remove any non-digit characters
+        $digits = preg_replace('/[^0-9]/', '', $number);
+
+        $code = CallingCode::fromPhoneNumber($number);
+        if ($code !== null) {
+            return substr($digits, strlen($code));
+        }
+
+        return $digits;
+    }
```

Comment:
Will it always be used with Indian numbers only?

Reply:
Yes currently fast2sms only supports indian phone numbers. However this function is generic and does not have any +91 specific overrides